db/migrate/20161223153249_add_permissionsto_arf_report.rb

```diff
@@ -0,0 +1,11 @@
+class AddPermissionstoArfReport < ActiveRecord::Migration
```

Comment:
s/AddPermissionstoArfReport/AddPermissionsToArfReport/

also the filename needs underscore